src/OnlineRetailPortal.Contracts/Models/GetProductsServiceRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace OnlineRetailPortal.Contracts
+{
+   public class GetProductsServiceRequest
+    {
+        public int PageNo { get; set; }
+        public int PageSize { get; set; }
+        public int TotalPages { get; set; }
+
```

Comment:
Similar to comment above. Class names should be Nouns like 'ProductsServiceRequest'